bid.go

```diff
@@ -20,33 +20,39 @@ var (
+	LossURL          string              `json:"lurl,omitempty"`           // Loss notice URL.
+	AdMarkup         string              `json:"adm,omitempty"`            // Actual ad markup. XHTML if a response to a banner object, or VAST XML if a response to a video object.
+	AdvDomains       []string            `json:"adomain,omitempty"`        // Advertiser’s primary or top-level domain for advertiser checking; or multiple if imp rotating.
+	Bundle           string              `json:"bundle,omitempty"`         // A platform-specific application identifier intended to be unique to the app and independent of the exchange.
+	ImageURL         string              `json:"iurl,omitempty"`           // Sample image URL.
+	CampaignID       StringOrNumber      `json:"cid,omitempty"`            // Campaign ID that appears with the Ad markup.
+	CreativeID       string              `json:"crid,omitempty"`           // Creative ID for reporting content issues or defects. This could also be used as a reference to a creative ID that is posted with an exchange.
+	Tactic           string              `json:"tactic,omitempty"`         // Tactic ID to enable buyers to label bids for reporting to the exchange the tactic through which their bid was submitted.
+	Categories       []ContentCategory   `json:"cat,omitempty"`            // IAB content categories of the creative. Refer to List 5.1
+	Attrs            []CreativeAttribute `json:"attr,omitempty"`           // Array of creative attributes.
+	API              APIFramework        `json:"api,omitempty"`            // API required by the markup if applicable, NOTE: for ORTB ver <= 2.5 APIFramework supported is 1 to 6.
+	APIS             APIFramework        `json:"apis,omitempty"`           // APIS required by the markup if applicable.
+	Protocol         Protocol            `json:"protocol,omitempty"`       // Video response protocol of the markup if applicable
+	MediaRating      IQGRating           `json:"qagmediarating,omitempty"` // Creative media rating per IQG guidelines.
+	Language         string              `json:"language,omitempty"`       // Language of the creative using ISO-639-1-alpha-2.
+	DealID           string              `json:"dealid,omitempty"`         // DealID extension of private marketplace deals
+	Width            int                 `json:"w,omitempty"`              // Width of the ad in pixels.
+	Height           int                 `json:"h,omitempty"`              // Height of the ad in pixels.
+	WidthRatio       int                 `json:"wratio,omitempty"`         // Relative width of the creative when expressing size as a ratio.
+	HeightRatio      int                 `json:"hratio,omitempty"`         // Relative height of the creative when expressing size as a ratio.
+	LangB            string              `json:"langb,omitempty"`          // Language of the creative using IETF BCP 47. Only one of language or langb should be present.
```

Comment:
Sorry, could you please append the new fields after a space? It is difficult to see in the diff what's actually changed. Thanks!

Reply:
Ahh sorry, I have added an extra new line before and after the new fields added.
Can you please check @dim ?
Thanks